addons/mail/static/src/discuss/call/common/thread_actions.js

```diff
@@ -29,13 +29,16 @@ threadActionsRegistry
     })
     .add("settings", {
         component: CallSettings,
+        componentProps(action) {
+            return { isCompact: true };
+        },
         condition(component) {
             return (
                 component.thread?.allowCalls &&
-                (!component.props.chatWindow || component.props.chatWindow.isOpen)
+                (component.props.chatWindow?.isOpen || component.store.inPublicPage)
             );
         },
-        panelOuterClass: "o-discuss-CallSettings",
+        panelOuterClass: "p-2",
```

Comment:
Conditionally on chat window, because in public page this is too much spacing and the background color is off:

![Screenshot 2024-06-28 at 11 59 33](https://github.com/odoo/odoo/assets/6569390/a899084e-d8a0-409c-9090-5b5081233efc)